closure/goog/fx/abstractdragdrop.js

```diff
@@ -319,8 +320,9 @@ goog.fx.AbstractDragDrop.prototype.initItem = function(item) {
 goog.fx.AbstractDragDrop.prototype.disposeItem = function(item) {
   if (this.isSource_) {
     goog.events.unlisten(
-        item.element, goog.events.EventType.MOUSEDOWN, item.mouseDown_, false,
-        item);
+        item.element,
+        [goog.events.EventType.MOUSEDOWN, goog.events.EventType.TOUCHSTART],
+        item.mouseDown_, false, item);
```

Comment:
I don't think this will work. You bind both MOUSEDOWN and TOUCHSTART to mouseDown_ listener, but mouseDown_ intentionally returns on TOUCHSTART. This will break basically everything as the "start drag" event will never fire. 

In general, this change needs to do some refactoring to abstract the concept of "mouse" out of  these methods.